robottelo/cli/base.py

```diff
@@ -252,6 +253,20 @@ def execute(
         else:
             return cls._handle_response(response, ignore_stderr=ignore_stderr)
 
+    @classmethod
+    def fm_execute(
+        cls,
+        command,
+        hostname=None,
+        timeout=None,
+    ):
+        """Executes the foreman-maintain cli commands on the server via ssh"""
+        from robottelo.ssh import get_client
```

Comment:
Regardless it's being used or not anywhere, all the imports should be initiated in the top of the module unless needed explicitely. That's the standard we are maintaining in robottelo.

And, I dont see that issue here in moving this to the top.

Reply:
Updated